api/src/main/java/org/confeti/dataobjects/Status.java

```diff
@@ -1,5 +1,10 @@
 package org.confeti.dataobjects;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
 public class Status {
```

Comment:
You can create enum without outer class [see](https://www.baeldung.com/jackson-serialize-enums)